docs/dataset/formatting-your-datasets.md

```diff
@@ -23,26 +23,34 @@ dataset with the following columns:
+|-------------|------------------------------------------------------------------------------------------------------------------------------------------|
+| `locator`     | Url path to a file to be displayed, either a [cloud storage](../connecting-cloud-storage/index.md) url or a http url that serves a file. |
+| `text`        | Raw text input for text based models.                                                                                                    |
 
-A `locator` is a url path to a file that will be displayed on the platform and can either be a
-[cloud storage](../connecting-cloud-storage/index.md) url or a http url that serves a file.
 A locator needs to have correct extensions for the corresponding file type. For example an image should be in a format
 such as `.jpg` or `.png`, whereas locators for audio data should be in forms like `.mp3` or `.wav`.
 
-| Data Type   | Supported file formats                                                                |
-|-------------|---------------------------------------------------------------------------------------|
-| Image       | `jpg`, `jpeg`, `png`, `gif`, `bmp` and other web browser supported image types.       |
-| Audio       | `flac`, `mp3`, `wav`, `acc`, `ogg`, `ra` and other web browser supported audio types. |
-| Video       | `mov`, `mp4`, `mpeg`, `avi` and other web browser supported video types.              |
-| Document    | `txt` and `pdf` files.                                                                |
-| Point Cloud | `pcd` files.                                                                          |
+The table below shows the supported file formats for the supported data types:
 
-For text-based models the `text` field contains the raw text input for the models.
+| Data Type      | Supported file formats                                                                |
+|----------------|---------------------------------------------------------------------------------------|
+| Image          | `jpg`, `jpeg`, `png`, `gif`, `bmp` and other web browser supported image types.       |
+| Audio          | `flac`, `mp3`, `wav`, `acc`, `ogg`, `ra` and other web browser supported audio types. |
+| Video          | `mov`, `mp4`, `mpeg`, `avi` and other web browser supported video types.              |
+| Document       | `txt` and `pdf` files.                                                                |
+| Point Cloud    | `pcd` files.                                                                          |
+
+By default the `locator` or  `text` fields are used if present in your dataset, and other fields
+can be specified when importing via the Web App from the [:kolena-dataset-16: Datasets](https://app.kolena.io/redirect/datasets)
+page, or the SDK by using [`upload_dataset`](../reference/dataset/index.md#kolena.dataset.dataset.upload_dataset)
+function.
```

Comment:
Not clear this is talking about inferring `id_fields`. Can we be more explicit here eg "Kolena will attempt to infer common id fields (eg. `locator`, `text`) based on what is present in the dataset during import. This can be overriden by explicitly declaring id fields."

Reply:
updated